### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by Onehour0108** [Xiyue-Wang/1st-in-MICCAI-MIDOG-2021-challenge#2]
你好，感谢你们提供的代码。我们对你们的工作很感兴趣。关于你们的方法，我们有下面的疑问需要你们的帮助：
由于本工程没有提供训练代码，我们按照文章提供的参数和细节搭建了一个训练框架以复现你们的训练过程，细节如下：
Patch size:512*512
Batch size:4
Learning rate:0.0003
Weight decay:0.0001
Adam optimizer
Loss: focalloss+diceloss
我们严格按照论文细节进行训练过程的复现，然而我们训练得到的结果并不正常。
请问训练代码是否还有额外需要注意的细节被我们忽略了，或者能否提供一下你们官方的训练代码帮助我们，期待你的回复。
